lib/mapObjects/CRewardableObject.h

```diff
@@ -104,9 +104,10 @@ class DLL_LINKAGE CRewardInfo
 	/// if set to true, object will be removed after granting reward
 	bool removeObject;
 
-	/// Generates list of components that describes reward
-	virtual void loadComponents(std::vector<Component> & comps) const;
-	Component getDisplayedComponent() const;
+	/// Generates list of components that describes reward for a specific hero
+	virtual void loadComponents(std::vector<Component> & comps,
+	                            const CGHeroInstance * h) const;
```

Comment:
To be more precise, here is the motivation: https://google.github.io/styleguide/cppguide.html#Line_Length